### ISSUE TITLE
Issue with absorbing boundary conditions?

### ISSUE THREAD

**Issue by PrometheusPi** [ComputationalRadiationPhysics/picongpu#1677]
While testing the new Laguerre Gauss laser (pull request #1580) I encountered an asymmetric boundary condition. In all dimension the absorbing boundary conditions are the same. But the absorption the at the start an end of the z-axis are stronger than along the x-axis. This leads to asymmetric laser pulses. 

Below you see an maximum of the absolute of the electric field (in polarization direction) along the propagation direction `np.amax(np.abs(Ex), axis=1)`

Gauss laser in x-polarized:
![ex](https://cloud.githubusercontent.com/assets/5121158/20975425/c9c5f234-bc9f-11e6-8209-7933d070f5cb.png)

and the same for a z-polarized Gauss laser:
![ez](https://cloud.githubusercontent.com/assets/5121158/20975475/e7f636c4-bc9f-11e6-9d47-03b5300e5d17.png)

Boundary conditions are:
```
    //! Defines the size of the absorbing zone (in cells)
    const uint32_t ABSORBER_CELLS[3][2] = {
        {48, 48},  /*x direction [negative,positive]*/
        {48, 48},  /*y direction [negative,positive]*/
        {48, 48}   /*z direction [negative,positive]*/
    }; //unit: number of cells

    //! Define the strength of the absorber for any direction
    const float_X ABSORBER_STRENGTH[3][2] = {
        {1.0e-3, 1.0e-3}, /*x direction [negative,positive]*/
        {1.0e-3, 1.0e-3}, /*y direction [negative,positive]*/
        {1.0e-3, 1.0e-3}  /*z direction [negative,positive]*/
    }; //unit: none

    BOOST_CONSTEXPR_OR_CONST uint32_t ABSORBER_FADE_IN_STEPS = 16;
```

cc'ing @BeyondEspresso and @ax3l

**Comment by ax3l** [comment]
we discussed this offline some months ago. do we need to investigate further with a simpler example?

**Comment by PrometheusPi** [comment]
Yes, we should continue to have a look into this problem. I avoided this issue by switching to periodic boundary conditions. (That's why I lost interest in having a detailed look into this asymmetry) But using periodic boundary conditions will not always be an option. Thus this will definitely pop up in the future again.

**Comment by ax3l** [comment]
@PrometheusPi this should be fixed with #2042 now?

**Comment by PrometheusPi** [comment]
@ax3l Yes - this solves this issue
